runtests.sh

```diff
@@ -118,6 +123,20 @@ fi
 # Unconditionally report on the state of monai
 python -c 'import monai; monai.config.print_config()'
 
+#Cleaning the files when ussing --clean in args and exit
+if [ "$clean_cache_files" = true ]
+then
+  rm -r mypy_cache
+
+  rm -r pytype
```

Comment:
we'll have a problem when `.mypy_cache` doesn't exist but `.pytype` exist, `rm -r mypy_cache` will raise an error, program exits, and the command `rm -r pytype` becomes not reachable.

Reply:
fixing that ASAP, i was able to run the project con windows but not on OS Catalina